developer_guide/installation/system_requirements/setup_requirements_ubuntu_1404.rst

```diff
@@ -22,7 +22,7 @@ Base dependencies
     $ sudo apt-get install php5-apcu
 
 .. note::
-    PHP 5.5 provided in Ubuntu 14.04 comes with the Zend OPcache opcode cache. Only the data cache provided by APCu is needed.
+    PHP 5.6 provided in Ubuntu 14.04 comes with the Zend OPcache opcode cache. Only the data cache provided by APCu is needed.
```

Comment:
Don't know what I have to do about the ubuntu and PHP version as Ubuntu 14.04 LTS provides 5.5 by default and Ubuntu 16.04 LTS provides PHP 7 by default.